modules/components/pages/outputs/cypher.adoc

```diff
@@ -0,0 +1,409 @@
+= cypher
+:type: output
+:status: experimental
+:categories: ["Services"]
+
+// © 2024 Redpanda Data Inc.
+
+
+component_type_dropdown::[]
+
+
+Writes a batch of messages to any graph database that supports the Neo4j or Bolt protocols.
```

Comment:
```suggestion
Writes a batch of messages to any graph database that supports the https://neo4j.com/docs/bolt/current/bolt/[Bolt protocol^] and the https://neo4j.com/docs/cypher-manual/current/introduction/[Cypher query language^]. For each incoming message, the connector can perform operations to store or delete data within the database using the Cypher query language.
```

@Jeffail please check this suggestion. I removed the reference to the Neo4j protocol because I couldn't find anything about it but I did see that [Neo4j supports the Bolt protocol as well as HTTP and HTTPS](https://neo4j.com/docs/operations-manual/current/configuration/connectors/). I also explicitly mention the Cypher query language because of the name of this connector.

Reply:
Or did we mean the Neo4j and Bolt URI schemes?

Reply:
@rockwotj. Could you confirm?

Reply:
Cypher is the query language, bolt is the "open standard" based protocol and AFAIK neo4j protocol adds clustering etc on top of bolt

Reply:
Updated.